frontend/src/lib/api.js

```diff
@@ -63,7 +63,7 @@ class Api {
         }
         return await getJSONOrThrow(response)
     }
-    async create(url, data) {
+    async create(url, data = undefined) {
```

Comment:
I think this _may_ cause an issue downstream? On line 77 do we need to verify that data isn't undefined before stringifying it it?

Reply:
`JSON.stringify(undefined)` returns `undefined` which would send an empty payload so I think this is fine, but not an expert on this part of the code.

Reply:
Merging this for now to fix this ASAP, but if you think this should be different feel free to add a follow-up PR 🙏